edith-step/src/UserChart.js

```diff
@@ -134,37 +127,49 @@ const BarGraph = () => {
             yAxes: [{
               scaleLabel: {
                 display: true,
+                fontSize: 20,
                 labelString: 'Dollars'
               },
               ticks: {
-                beginAtZero: true
+                beginAtZero: true,
               }
             }],
             xAxes: [{
               scaleLabel: {
                 display: true,
+                fontSize: 20,
                 labelString: 'Week'
-              }
+              },
             }],
           }    
         }}
       />
     );
-  }
+}
 
 
-const PieChart = () => {
+const DoughnutChart = (props) => {
     
   const [chartData, setChartData] = useState({});
   const chart = () => {
-    axios
-      .get("/user-stats-servlet")
-      .then(response => {
-        console.log(response);
+    fetch("/user-stats-servlet")
+      .then(response => response.json())
+      .then(responseJson => {
```

Comment:
nit: parentheses on (response) and (responseJson) too

Reply:
I changed this in a later commit.